src/test/resources/clear.sql

```diff
@@ -1,12 +1,21 @@
-DELETE from answer;
-DELETE from agreement;
-DELETE from petition;
+DELETE
+from answer;
+DELETE
+from agreement;
+DELETE
+from petition;
 
-DELETE from sign_up_verification_info;
-DELETE from password_verification_info;
-DELETE from user;
+DELETE
+from sign_up_verification_info;
+DELETE
+from password_verification_info;
+DELETE
+from user;
 
-DELETE from answer_aud;
-DELETE from petition_aud;
-DELETE from revinfo;
+DELETE
+from answer_aud;
+DELETE
+from petition_aud;
+DELETE
+from revinfo;
```

Comment:
이건 굳이 이렇게 두 줄로 한 이유가 있나요...?
```suggestion
DELETE FROM -
```

Reply:
자동 포맷팅이 저렇게 하더라구요 ..